tsdb/chunks/head_chunks_test.go

```diff
@@ -118,11 +118,6 @@ func TestChunkDiskMapper_WriteChunk_Chunk_IterateChunks(t *testing.T) {
 	fileEnd := HeadChunkFileHeaderSize + len(expectedBytes)
 	require.Equal(t, expectedBytes, actualBytes[HeadChunkFileHeaderSize:fileEnd])
 
-	// Test for the next chunk header to be all 0s. That marks the end of the file.
-	for _, b := range actualBytes[fileEnd : fileEnd+MaxHeadChunkMetaSize] {
```

Comment:
I double-checked and we don't need this assertion. If anything is wrong here (some of those bytes are non-zero), `IterateAllChunks` will fail with `CorruptionErr` anyway: https://github.com/prometheus/prometheus/blob/0adfa7cbed084209b4f582bbc3610244a3471b56/tsdb/chunks/head_chunks.go#L574